examples/examples/balance_transfer.rs

```diff
@@ -46,7 +46,7 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
         .to_runtime_api::<polkadot::RuntimeApi<DefaultConfig, DefaultExtra<DefaultConfig>>>();
     let hash = api
         .tx()
-        .balances()
+        .balances()?
```

Comment:
It'll be interesting to see how you implement the use case where I'd like to ignore any checks for a given pallet and go ahead anyway!

Reply:
Also, do you still have in mind to have a global hash that can warn if _anything_ in the metadata is different?